gcloud-java-datastore/src/test/java/com/google/gcloud/datastore/DatastoreTest.java

```diff
@@ -691,6 +696,37 @@ public void testQueryPaginationWithLimit() throws DatastoreException {
   }
 
   @Test
+  public void testEventuallyConsistentQuery() {
+    DatastoreRpcFactory rpcFactoryMock = EasyMock.createStrictMock(DatastoreRpcFactory.class);
+    DatastoreRpc rpcMock = EasyMock.createStrictMock(DatastoreRpc.class);
+    EasyMock.expect(rpcFactoryMock.create(EasyMock.anyObject(DatastoreOptions.class)))
+        .andReturn(rpcMock);
+    com.google.datastore.v1beta3.ReadOptions readOption =
```

Comment:
I can't do it for GqlQuery (b/c the name collides with one of our own classes), but I will make changes to import classes without name collisions.